tests/interfaces/test_autograd_qnode_new.py

```diff
@@ -191,9 +199,7 @@ def test_jacobian(self, interface, dev_name, diff_method, grad_on_execution, moc
 
         dev = qml.device(dev_name, wires=num_wires)
 
-        @qnode(
-            dev, diff_method=diff_method, interface=interface, grad_on_execution=grad_on_execution
-        )
+        @qnode(dev, **kwargs)
         def circuit(a, b):
```

Comment:
@albi3ro Do these changes look good to you? There are a lot more of them, but not all of them are exactly the same.

Reply:
Changes are looking good.

Setting a seed in the tests is good for avoiding intermittent failures and reproducible results, but I don't think we need to strictly set it in all of the tests. If we don't set the seed, `spsa` just creates its own random rng, which is perfectly fine.